src/pages/iou/request/step/IOURequestStepConfirmation.tsx

```diff
@@ -78,6 +78,21 @@ function IOURequestStepConfirmation({
     const isSharingTrackExpense = action === CONST.IOU.ACTION.SHARE;
     const isCategorizingTrackExpense = action === CONST.IOU.ACTION.CATEGORIZE;
     const isSubmittingFromTrackExpense = action === CONST.IOU.ACTION.SUBMIT;
+    const payeePersonalDetails = useMemo(() => {
+        if (personalDetails?.[transaction?.splitPayerAccountIDs?.[0] ?? -1]) {
+            return personalDetails?.[transaction?.splitPayerAccountIDs?.[0] ?? -1];
+        }
+
+        const participant = transaction?.participants?.find((val) => val.accountID === (transaction?.splitPayerAccountIDs?.[0] ?? -1));
```

Comment:
I doubt there's a scenario where this code would run. Have you ran into an issue without it?

Reply:
@youssef-lr This happened when we search a new user in participant page and then select it as split payer. The personal detail is empty so I created an optimistic personal detail for it.

Reply:
Hm. Do we not create an entry for new users when we search for them? Can you try searching for a new email, then once it's displayed check personal details in Onyx if we added them?

Reply:
When we search for a user that we haven't had the chat in offline or new users that doesn't have Expensify account, we don't have personal detail of this user in Onyx.

Reply:
Ah ok, I was under the impression this [code](https://github.com/Expensify/App/blob/058044b0baa307afb5325c8ecdcd7cae9357d03e/src/libs/OptionsListUtils.ts#L1860) saves the optimistic details